src/activities/components/ActivityList.vue

```diff
@@ -2,8 +2,20 @@
   <div>
     <div
       v-if="filter && activities.length > 0"
-      class="row no-wrap items-center justify-between bg-white q-px-sm q-py-xs"
+      class="row no-wrap items-center justify-between bg-white q-px-sm q-py-xs q-gutter-sm"
     >
+      <span v-t="'ACTIVITYLIST.TYPE'" />
+      <QSelect
+        v-model="type"
+        :options="typeOptions"
+        emit-value
+        map-options
+        outlined
+        hide-bottom-space
+        dense
+      />
+
```

Comment:
If activity list is empty, show something?

Reply:
I made it do this:

![activityfilternoresults](https://user-images.githubusercontent.com/31616/97777644-ed372280-1b71-11eb-8a37-80db708b285e.png)

The message is not 100% perfect, but it reuses the existing one, if there are no activities at all (rather than just filtered out).